app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -105,17 +108,28 @@ private fun HomeSectionHeaderContent(
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
-                    .semantics {
-                        contentDescription = description
+            TextButton(
+                text = stringResource(id = R.string.recent_tabs_show_all),
+                shouldUppercaseText = false,
+                buttonColors = textButtonColors(
+                    contentColor = if (isSystemInDarkTheme()) {
+                        PhotonColors.White
+                    } else {
+                        PhotonColors.Black
                     },
-                style = TextStyle(
-                    color = showAllTextColor,
-                    fontSize = 14.sp,
                 ),
                 onClick = { onShowAllClick() },
+                modifier = Modifier
+                    .semantics { contentDescription = description },
+                contentPaddingValues = PaddingValues(
+                    start = 16.dp,
+                    end = ButtonDefaults.TextButtonContentPadding.calculateEndPadding(
+                        LocalLayoutDirection.current,
+                    ),
+                    top = ButtonDefaults.TextButtonContentPadding.calculateTopPadding(),
+                    bottom = ButtonDefaults.TextButtonContentPadding.calculateBottomPadding(),
+                ),
```

Comment:
after switching from `ClickableText` to `TextButton`, the `contentPaddingValues` should no longer be necessary. I've tested this locally, and the clickable area looks fine without this.